libc/src/errno/libc_errno.cpp

```diff
@@ -1,40 +1,44 @@
-//===-- Implementation of errno -------------------------------------------===//
+//===-- Implementation of libc_errno --------------------------------------===//
 //
 // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
 
-#include "src/__support/macros/attributes.h"
```

Comment:
Can we keep these includes in this file even if they are already in libc_errno.h ?

Reply:
Done